### PR TITLE
feat(new notification): SA/add "target" prop to link {}

### PR DESCRIPTION
**Pull request by ScottAgirs**
### What does it do?

If merged, this will add more granular control over the link in the New Notification, namely provide the ability to _optionally_ provide `target` to the `link {}`. 

### Why is it needed?

It may be preferred in some cases not to open a new tab when clicking on the link.
For example: when a plugin does something where the user could benefit from navigating to a specific page, at the same time where hard-coding a programmatical redirect may not be optimal

### How to test it?

Use as you normally would, and add `target`. In the example to prevent opening a new tab, provide `_self`. Default to the current value - `_blank` - so no breaking changes.

```javascript
strapi.notification.toggle({
      type: 'Success',
      message: 'Boy! That escaled quickly',
      link: {
          target: '_self',
          label: 'Channel 4 News',
          url: 'https://anchorman.fandom.com/wiki/Channel_4_News_Team',
      },
});
```

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

Additionally, this PR includes a minuscule typo fix 😅

### CODE REVIEW

File: packages/strapi-admin/admin/src/containers/NewNotification/Notification/index.js [hunk 1]
```diff
@@ -100,8 +108,12 @@ const Notification = ({ notification }) => {
                       >
                         {formattedMessage(link.label)}
                       </Text>
-                      <Padded left size="xs" />
-                      <LinkArrow />
+                      {link.target === "_blank" && (
+                        <>
+                          <Padded left size="xs" />
+                          <LinkArrow />
+                        </>
```
Comment (HichamELBSI) [inline packages/strapi-admin/admin/src/containers/NewNotification/Notification/index.js:115]: Can you wrap the LinkArrow with the Padded component and remove the empty tags please 😉 .

```js
<Padded left size="xs">
   <LinkArrow />
</Padded>
```

**Review by HichamELBSI — CHANGES_REQUESTED** [review]
Hello @ScottAgirs, Thanks for the contribution 😄

File: packages/strapi-admin/admin/src/containers/NewNotification/Notification/index.js (hunk 1 above)
Comment (ScottAgirs) [reply packages/strapi-admin/admin/src/containers/NewNotification/Notification/index.js]: Whaha, true my bad, done!

**Review by soupette — APPROVED** [review]
LGTM.

Thank you for your contribution!

**Review by HichamELBSI — APPROVED** [review]
LGTM thanks :) can you update the branch please so that, we can merge this one?

**Comment by ScottAgirs** [comment]
> LGTM thanks :) can you update the branch please so that, we can merge this one?

Done 😃

**Pull request merged**